### ISSUE TITLE
(bug) The app window is not opening when starting the second app instance

### ISSUE THREAD

**Issue by stenya** [ivpn/desktop-app#98]
# Bug report

## Describe your environment

* Device: Windows     
* IVPN app version: 3.4.4

### Steps to reproduce:

1. Minimize to tray - enabled; 
2. Start the app -> app is started; tray icon appeared; UI is not shown
3. Try to start the second app instance (e.g. clicking app link from Windows start menu)

### Observed Results:

App UI not shown. 

### Expected Results:

The app window should be shown

**Comment by stenya** [comment]
v3.4.5

**Comment by stenya** [comment]
UPD: the issue related to Linux version also

**Comment by gorkapernas** [comment]
Verified fixed on version 3.4.5, tested on Windows 10/11 and Linux - Ubuntu and Fedora, 
I was able to reproduce the issue first on the previous version on multiple systems, but on version 3.4.5 the issue no longer occurs. When the UI is minimized to tray, on Windows, the UI can be opened from the desktop shortcut or the start menu, on Linux, the UI can be opened from the applications menu. 
This is good to go.
